src/doc/rustc/src/platform-support.md

```diff
@@ -190,7 +190,8 @@ target | std | notes
 [`thumbv8m.main-none-eabihf`](platform-support/arm-none-eabi.md) | * | Bare ARMv8-M Mainline, hardfloat
 `wasm32-unknown-emscripten` | ✓ | WebAssembly via Emscripten
 `wasm32-unknown-unknown` | ✓ | WebAssembly
-`wasm32-wasi` | ✓ | WebAssembly with WASI
+`wasm32-wasi` | ✓ | WebAssembly with WASI (undergoing a [rename][wasi-rename])
```

Comment:
nit: perhaps it would be good to go ahead and say "undergoing a rename to `wasm32-wasi-preview1`" and save users a click

Reply:
Heh good point! Rebased and updated